<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInfo</class>
 <widget class="QWidget" name="UserInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="headBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UIDLbl">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>   UID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLbl">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>   name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="scoreLbl">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>   score:</string>
   </property>
  </widget>
  <widget class="QLabel" name="moneyLbl">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>   money:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
